.github/pull_request_template.md

```diff
@@ -16,9 +16,9 @@
 
 <!--
     In case of bug fixes, please provide the list of supported branches where this fix should be also merged.
-    Please uncomment following line with the corresponding branches.
+    Please uncomment following line, adjusting the corresponding target branches for the backport.
 -->
-<!-- @Mergifyio backport (branch/es) -->
+<!-- @Mergifyio backport 2.8.x 2.7.x 2.6.x 2.1.x -->
```

Comment:
2.9.x must be included too. Current master is already aligned with the, not yet release, 2.10.x branch